advancedTodo/src/redux/todoAppComplete/todoConstants.js

```diff
@@ -1,7 +1,10 @@
 export const ADD_TODO = 'ADD_TODO';
-export const DELETE_TODO= 'DELETE_TODO';
+export const DELETE_TODO = 'DELETE_TODO';
 export const COMPLETE_TODO = 'COMPLETE_TODO';
 export const COMPLETED = 'completed';
 export const SAVE_DESCRIPTION = 'SAVE_DESCRIPTION';
-export const SHOW_DESCRIPTION= 'SHOW_DESCRIPTION';
-export const CREATE_NOTIFICATION= 'CREATE_NOTIFICATION';
\ No newline at end of file
+export const SHOW_DESCRIPTION = 'SHOW_DESCRIPTION';
+export const CREATE_NOTIFICATION = 'CREATE_NOTIFICATION';
+export const SUCCESS_MESSAGE = "Information successfully saved', 'Congratulations !"
+export const DELETE_MESSAGE = "Todo Deleted Successfully', 'Congratulations !"
+export const TODO_COMPLETED_MESSAGE = "Todo Completed Successfully', 'Congratulations !"
```

Comment:
This code defines notification messages. Consider adding JSDoc comments to describe the purpose of each constant and the message it represents. This will improve code readability and understanding.